people_tool/static/people_tool/js/controllers/SearchController.js

```diff
@@ -98,9 +98,23 @@
             // can still display a clickable anchor link
             var lastName = (full.name_last.trim().length > 0) ?
                 full.name_last : '(none)';
-            return '<a href="' + '#/people/' + full.univ_id + '/courses/">'
+            return '<a href="' + '#/people/' + full.univ_id + '/courses/" ' +
+                'ng-click="setSelectedPersonInfo(\''+full.name_last+'\',\''+
+                full.name_first+'\','+full.univ_id+',\''+full.email_address+'\''+',\''+full.role_type_cd+'\')">'
                 + lastName + '</a>';
         };
+        $scope.setSelectedPersonInfo =
+            function(name_last, name_first, univ_id, email, role_type_cd){
```

Comment:
I tried passing the row's full object as a first pass - doesn't work :-( . Hence the passing in as separate params..